src/types.mligo

```diff
@@ -289,44 +320,10 @@ type initial_data =
 
 type decision_callback = decision_callback_input -> decision_callback_output
 
-type config =
-  { max_quorum_threshold : quorum_fraction
-  // ^ Determine the maximum value of quorum threshold that is allowed.
-  ; min_quorum_threshold : quorum_fraction
-  // ^ Determine the minimum value of quorum threshold that is allowed.
-
-  ; period : period
-  // ^ Determines the stages length.
-
-  ; fixed_proposal_fee_in_token : nat
-  // ^ A base fee paid for submitting a new proposal.
-
-  ; max_quorum_change : quorum_fraction
-  // ^ A percentage value that limits the quorum_threshold change during
-  // every update of the same.
-  ; quorum_change : quorum_fraction
-  // ^ A percentage value that is used in the computation of new quorum
-  // threshold value.
-  ; governance_total_supply : nat
-  // ^ The total supply of governance tokens used in the computation of
-  // of new quorum threshold value at each stage.
-
-  ; proposal_flush_level : blocks
-  // ^ The proposal age at (and above) which the proposal is considered flushable.
-  // Has to be bigger than `period * 2`
-  ; proposal_expired_level : blocks
-  // ^ The proposal age at (and above) which the proposal is considered expired.
-  // Has to be bigger than `proposal_flush_time`
-  }
-
-type full_storage = storage * config
```

Comment:
AFAICT the specification needs to be updated, because it still mentions this type.

Reply:
@sras AFAICT the spec still needs to be updated.

Other than that I don't have other comments.

Reply:
Yes, missed it. Changed now.